src/components/lib/pci/io.h

```diff
@@ -0,0 +1,45 @@
+#ifndef IO_H
+#define IO_H
+
+#include <cos_types.h>
+
```

Comment:
This is all going to break on non-x86. Any preferences on how to handle it?

@WenyuanShao @msdx321 @betahxy

Reply:
Maybe add a way to do conditional compilation of libraries/components into the build system? I don't think that we want this component, nor the test component in non-x86.